README.md

```
@@ -133,22 +131,20 @@ holiday-cmatrix -f my-custom-themes
+Create a text file with one theme per line and tab-separated fields:
 
 ```bash
 # Comments start with #
-# Format: theme_name<TAB>date_spec<TAB>cmatrix_arguments
-# Date spec: MM-DD or MM-DD:MM-DD
-# Comments are stripped from lines (everything after #)
+# Format: Name<TAB>Start: MM-DD[<TAB>End: MM-DD]<TAB>cmatrix_arguments
 
 # Override Christmas with custom magenta theme
-Custom Christmas	12-20:12-25	-a -C magenta,cyan,white -u 1 -B
+Christmas	12-20	12-25	-a -C magenta,cyan,white -u 1 -B
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 10

(MD010, no-hard-tabs)